backend/Application/Api/GraphQL/Import/IPendingBakerChangeStrategy.cs

```diff
@@ -92,13 +92,24 @@ public async Task<PendingBakerChange> SetPendingChangeOnBaker(BakerRemoved baker
     {
         var updatedBaker = await _writer.UpdateBaker(bakerRemoved, src => src.BakerId,
             (src, dst) => SetPendingChange(dst, src));
+        if (updatedBaker is null)
+        {
+            return null;
+        }
+        
         return ((ActiveBakerState)updatedBaker.State).PendingChange!;
     }
 
     public async Task<PendingBakerChange> SetPendingChangeOnBaker(BakerStakeDecreased stakeDecreased)
     {
         var updatedBaker = await _writer.UpdateBaker(stakeDecreased, src => src.BakerId,
             (src, dst) => SetPendingChange(dst, src));
+
+        if (updatedBaker is null)
+        {
+            return null;
+        }
```

Comment:
Do we want to log something here? Do we not always expect a non-null baker?

Reply:
this is being handled by the caller of the method.